Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Image-Cropper/index.md

```diff
@@ -64,17 +64,28 @@ For rendering a cropped media item, the `.GetCropUrl` is used:
 ### MVC View Example to output a "banner" crop from a cropper property with the alias "image"
 
 ```html
-<img src="@Url.GetCropUrl(Model.Image, "banner")" />
+<img src="@(Url.GetCropUrl(Model.Image, "banner"))" />
 ```
 
 Or, alternatively:
+```html
+<img src="@(Model.Image.GetCropUrl("banner", Current.ImageUrlGenerator))" />
+```
 
+This was obsoleted in 8.7.0:
 ```html
-<img src="@Model.Image.GetCropUrl("banner")" />
+<img src="@(Model.Image.GetCropUrl("banner"))" />
 ```
 
 ### MVC View Example to output create custom crops - in this case forcing a 300 x 400 px image
+```csharp
+@if (Model.HasValue("image"))
+{
+    <img src="@Model.Image.GetCropUrl(height: 300, width: 400, imageUrlGenerator: Current.ImageUrlGenerator)" />
+}
+```
 
+This was obsoleted in 8.7.0:
```

Comment:
If it was obsoleted, I think it would make sense to simply remove it from this article.
It will still be in the one for the version where the snippet actually applies.

What do you think?

Reply:
I wasn't sure about marking it as obsoleted, am more than happy to remove and have done so :-)